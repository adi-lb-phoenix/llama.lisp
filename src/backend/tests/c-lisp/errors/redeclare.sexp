(c-lisp
    (define ((main void))
        (declare var int)

        (if #t
            ((declare var float)
             (set var 0.5)))

        (declare var bool)))
