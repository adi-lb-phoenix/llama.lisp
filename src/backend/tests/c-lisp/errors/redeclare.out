Error in statement: ['declare', 'var', 'bool']
CodegenError: Re-declaration of variable var
